### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Ashokaas** [Snipeur060/Ecole-directe-classe#16] [Ideas]
Salut tu pourrais utiliser getpass pour demander le mdp pour que ça soit plus sécurisé, ça permet de ne pas afficher le mdp lors de la saisie
Si tu connais pas c'est pas compliqué, 
```
from getpass import getpass
mdp = getpass('Votre mot de passe : ')
```
Bonne journée :)

**Comment by Snipeur060** [comment]
Salut je vais ajouter et t'ajouter dans la pull request :-)

**Comment by Snipeur060** [comment]
C'est ajouté

**Reply by Snipeur060** [reply]
Je n'ai pas mis getpass car elle ne contient pas le stream de * à la place du mdp
